### PR TITLE
[#10] As a user, I can see the list of TILs

### PR DESCRIPTION
**Pull request by hoangmirs**
Resolves https://github.com/nimblehq/til-web/issues/10

## What happened 👀

- Install TailwindCSS and DaisyUI. Then configuring postCSS to work with them.
- Create a layout for pages.
- Render TIL list

## Insight 📝

- As we have not had a design yet, then adding DaisyUI to use some pre-built components
- Fix stylelint command to run only `src` folder

## Proof Of Work 📹

[You can check here](https://deploy-preview-19--nimble-til.netlify.app/)

### CODE REVIEW

File: src/components/Layout/index.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  children: React.ReactNode;
+}
+
+const Layout = ({ children }: LayoutProps) => {
+  return (
+    <>
+      <Head>
+        <title>TIL</title>
+        <meta name="description" content="What Nimble has learned today" />
+        <link rel="icon" href="/favicon.ico" />
```
Comment (Lahphim) [inline src/components/Layout/index.tsx:15]: Should we move this to the global `_document.page.tsx`?

```
import { Html, Head, Main, NextScript } from 'next/document';

function Document() {
  return (
    <Html lang="en">
      <Head>
        <link rel="icon" href="/favicon.ico" />
      </Head>
      <body>
        <Main />
        <NextScript />
      </body>
    </Html>
  );
}

export default Document;
```

File: src/components/Post/List.module.css [hunk 2]
```diff
@@ -0,0 +1,3 @@
+.listContainer {
```
Comment (Lahphim) [inline src/components/Post/List.module.css:1]: Should we omit this file as we don't use it?

File: src/components/Layout/index.tsx [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+      </Head>
+
+      <main className={styles.mainContainer}>{children}</main>
```
Comment (tyrro) [inline src/components/Layout/index.tsx:18]: I am just curious why we are not applying the utility classes directly here? Using the `@apply` directive requires us to come up with class names which tailwind [discourages](https://tailwindcss.com/docs/utility-first) firmly?

**Review by tyrro — APPROVED** [review]

File: src/components/Post/List.module.css (hunk 2 above)
Comment (hoangmirs) [reply src/components/Post/List.module.css:1]: Yes, we should

File: src/components/Layout/index.tsx (hunk 1 above)
Comment (hoangmirs) [reply src/components/Layout/index.tsx]: Added in https://github.com/nimblehq/til-web/pull/19/commits/e39e645e37eb1a5805164aa818733149422375ff

File: src/components/Post/List.module.css (hunk 2 above)
Comment (hoangmirs) [reply src/components/Post/List.module.css]: I removed CSS modules files and inject the utilities classes to React component in https://github.com/nimblehq/til-web/pull/19/commits/455f1315063731395c5f71e0017c7895733cbc06

**Review by Lahphim — APPROVED** [review]

**Pull request merged**